system/HTTP/CURLRequest.php

```diff
@@ -39,13 +39,6 @@ class CURLRequest extends OutgoingRequest
      */
     protected $responseOrig;
 
-    /**
-     * The URI associated with this request
-     *
-     * @var URI
-     */
-    protected $baseURI;
```

Comment:
bc break.

Reply:
Excuse me, this is my first time to submit pull request. What's meaning 'bc break'? And what should I do next?

Reply:
backward compatibility break, remove protected property is not allowed on fix pr.

Reply:
Thanks! Maybe I should submit to next version.